example-mapfragmentandview/SampleMapViewPgm/src/main/java/mil/emp3/examples/samplemapviewpgm/MainActivity.java

```diff
@@ -142,14 +136,11 @@ public void onEvent(MapStateChangeEvent mapStateChangeEvent) {
         } catch (EMP_Exception e) {
             Log.e(TAG, "addMapStateChangeEventListener", e);
         }
+
         if (!restartingActivity) {
             try {
                 map2.swapMapEngine(properties);
-
-                final ICamera c = new Camera(); //TODO remove me thx
-                c.setAltitude(1e7);
-                map2.setCamera(c, false);
-                
+                map2.setCamera(CameraUtility.buildCamera(40.7128, -74.0059, 2000000.0), false);
```

Comment:
It can be any value. I just selected east and west coast for map1/map2 respectively.